eventio/simtel/objects.py

```diff
@@ -37,6 +38,20 @@ class SimTelRunHeader(EventIOObject):
 
 class SimTelMCRunHeader(EventIOObject):
     eventio_type = 2001
+    from .mc_runheader_dtypes import mc_runheader_dtype_map
+
+    def parse_data_field(self):
+        ''' '''
+        self.seek(0)
+        data = self.read()
+
+        header_type = self.mc_runheader_dtype_map[self.header.version]
```

Comment:
Maybe raise a nicer error message than keyerror if there is an unsupported version.